gradle.properties

```diff
@@ -1,3 +1,4 @@
 ossrhUsername=scenarioo
 # Set the password in your user home gradle.
-ossrhPassword=
\ No newline at end of file
+ossrhPassword=
+org.gradle.jvmargs=-Xmx4096m
```

Comment:
Do we really want to enforce this so high - will this work on CI??

See what we have in circleci config file